### DISCUSSION TITLE
[please disregard, I'll ask again after troubleshooting more]

### DISCUSSION THREAD

**Discussion by floer32** [TanStack/virtual#369] [Q&A] [answered]
Edit: I'm trying to reproduce the issue in a CodeSandbox, will open a new thread when I have.

**Comment by floer32** [comment, marked as answer]
[closing]

**Reply by ardeora** [reply]
> Edit: I'm trying to reproduce the issue in a CodeSandbox, will open a new thread when I have.

Thanks! Sorry I couldn’t take a look at it today. Whenever you have a reproduction ready. Let me know and I’ll try to dig in. It’s totally possIble there are a few bugs that need to be ironed out 🙂

**Reply by floer32** [reply]
Oh, no problem at all! Thanks!
